pkg/loop/internal/relayer.go

```diff
@@ -168,6 +168,7 @@ type Relayer interface {
 	NewConfigProvider(context.Context, types.RelayArgs) (types.ConfigProvider, error)
 	NewMedianProvider(context.Context, types.RelayArgs, types.PluginArgs) (types.MedianProvider, error)
 	NewMercuryProvider(context.Context, types.RelayArgs, types.PluginArgs) (types.MercuryProvider, error)
+	NewFunctionsProvider(context.Context, types.RelayArgs, types.PluginArgs) (types.FunctionsProvider, error)
```

Comment:
🥳 

+1 to jordan's note on this will need to implemented into all the chain-specific repos otherwise core builds will fail. let me know if you need help with getting those PRs through